examples/virtio/Makefile

```diff
@@ -1,384 +1,64 @@
-LINUX_BLK_BOARD_DIR := $(LINUX_BLK_DIR)/board/$(BOARD)
-
-UIO_DIR := $(LINUX_DIR)/uio
-UIO_DRIVER_BLK_DIR := $(LINUX_DIR)/uio_drivers/blk
-
-SDDF_DIR := $(abspath ../../dep/sddf)
-
-VMM_INCLUDE := include
-
-BLK_DRIVER_VM_DIR := board/$(BOARD)/blk_driver_vm
-BLK_DRIVER_VM_DTS_DIR := $(BLK_DRIVER_VM_DIR)/dts
-BLK_DRIVER_VM_ROOTFS_TMP_DIR := $(BUILD_DIR)/blk_driver_vm_rootfs
-
-CLIENT_VM_1_DIR := board/$(BOARD)/client_vm_1
-CLIENT_VM_1_DTS_DIR := $(CLIENT_VM_1_DIR)/dts
-CLIENT_VM_1_ROOTFS_TMP_DIR := $(BUILD_DIR)/client_vm_1_rootfs
-
-CLIENT_VM_2_DIR := board/$(BOARD)/client_vm_2
-CLIENT_VM_2_DTS_DIR := $(CLIENT_VM_2_DIR)/dts
-CLIENT_VM_2_ROOTFS_TMP_DIR := $(BUILD_DIR)/client_vm_2_rootfs
-
-
-BLK_DRIVER_VM_LINUX := $(BLK_DRIVER_VM_DIR)/linux
-BLK_DRIVER_VM_INITRD := $(BLK_DRIVER_VM_DIR)/rootfs.cpio.gz
-BLK_DRIVER_VM_INITRD_MODIFIED := $(BUILD_DIR)/blk_driver_vm_rootfs.cpio.gz
-BLK_DRIVER_VM_BASE_DTS := $(BLK_DRIVER_VM_DTS_DIR)/linux.dts
-BLK_DRIVER_VM_DTS_OVERLAYS_qemu_arm_virt :=	$(BLK_DRIVER_VM_DTS_DIR)/init.dts \
-											$(BLK_DRIVER_VM_DTS_DIR)/io.dts \
-											$(BLK_DRIVER_VM_DTS_DIR)/disable.dts
-BLK_DRIVER_VM_DTS_OVERLAYS_odroidc4 :=	$(BLK_DRIVER_VM_DTS_DIR)/init.dts \
-										$(BLK_DRIVER_VM_DTS_DIR)/io.dts \
-										$(BLK_DRIVER_VM_DTS_DIR)/disable.dts
-BLK_DRIVER_VM_DTS_OVERLAYS := ${BLK_DRIVER_VM_DTS_OVERLAYS_${BOARD}}
-BLK_DRIVER_VM_DTB := $(BUILD_DIR)/blk_driver_vm.dtb
-
-
-CLIENT_VM_1_LINUX := $(CLIENT_VM_1_DIR)/linux
-CLIENT_VM_1_INITRD := $(CLIENT_VM_1_DIR)/rootfs.cpio.gz
-CLIENT_VM_1_INITRD_MODIFIED := $(BUILD_DIR)/client_vm_1_rootfs.cpio.gz
-CLIENT_VM_1_BASE_DTS := $(CLIENT_VM_1_DTS_DIR)/linux.dts
-CLIENT_VM_1_DTS_OVERLAYS_qemu_arm_virt :=	$(CLIENT_VM_1_DTS_DIR)/init.dts \
-											$(CLIENT_VM_1_DTS_DIR)/virtio.dts
-CLIENT_VM_1_DTS_OVERLAYS_odroidc4 :=	$(CLIENT_VM_1_DTS_DIR)/init.dts \
-										$(CLIENT_VM_1_DTS_DIR)/virtio.dts\
-										$(CLIENT_VM_1_DTS_DIR)/disable.dts
-CLIENT_VM_1_DTS_OVERLAYS := ${CLIENT_VM_1_DTS_OVERLAYS_${BOARD}}
-CLIENT_VM_1_DTB := $(BUILD_DIR)/client_vm_1.dtb
-
-
-CLIENT_VM_2_LINUX := $(CLIENT_VM_2_DIR)/linux
-CLIENT_VM_2_INITRD := $(CLIENT_VM_2_DIR)/rootfs.cpio.gz
-CLIENT_VM_2_INITRD_MODIFIED := $(BUILD_DIR)/client_vm_2_rootfs.cpio.gz
-CLIENT_VM_2_BASE_DTS := $(CLIENT_VM_2_DTS_DIR)/linux.dts
-CLIENT_VM_2_DTS_OVERLAYS_qemu_arm_virt :=	$(CLIENT_VM_2_DTS_DIR)/init.dts \
-											$(CLIENT_VM_2_DTS_DIR)/virtio.dts
-CLIENT_VM_2_DTS_OVERLAYS_odroidc4 :=	$(CLIENT_VM_2_DTS_DIR)/init.dts \
-										$(CLIENT_VM_2_DTS_DIR)/virtio.dts \
-										$(CLIENT_VM_2_DTS_DIR)/disable.dts
-CLIENT_VM_2_DTS_OVERLAYS := ${CLIENT_VM_2_DTS_OVERLAYS_${BOARD}}
-CLIENT_VM_2_DTB := $(BUILD_DIR)/client_vm_2.dtb
-
-SDDF_BLK_COMPONENTS := $(SDDF_DIR)/blk/components
-SDDF_BLK_UTIL := $(SDDF_DIR)/blk/util
-
-ifeq ($(strip $(BOARD)), odroidc4)
-	export UART_DRIVER_DIR := meson
-else ifeq ($(strip $(BOARD)), qemu_arm_virt)
-	export UART_DRIVER_DIR := arm
+export override MICROKIT_SDK:=$(abspath $(MICROKIT_SDK))
```

Comment:
Move into virtio.mk

Reply:
Offline discussion: going to put this in another PR.